internal/context/repo.go

```diff
@@ -96,6 +97,22 @@ func (r *Repository) GetEditorconfig() (*editorconfig.Editorconfig, error) {
 	return editorconfig.ParseBytes(data)
 }
 
+// MakeURL gets string or url.URL as argument and returns URL prepended with repository URL
+func (r *Repository) MakeURL(location interface{}) string {
+	switch location := location.(type) {
+	case string:
+		tempURL := url.URL{
+			Path: r.RepoLink + "/" + location,
+		}
+		return tempURL.String()
+	case url.URL:
+		location.Path = r.RepoLink + "/" + location.Path
+		return location.String()
+	default:
+		return r.RepoLink
```

Comment:
I think it should panic, swallow it might cause hard to find bugs.
```suggestion
		panic("location type must be either string or url.URL")
```